its/ruling/src/test/resources/eclipse-jetty/java-S6212.json

```diff
@@ -0,0 +1,4066 @@
+{
```

Comment:
I checked nearly 500 issues. The majority of them where TP, some of them where FP to me, so I updated the rule implementation with some constraints, where it's not possible/not recommended to use "var"